charts/sentry/values.yaml

```diff
@@ -0,0 +1,939 @@
+      enabled: true
+      dataPersistentVolume:
+        enabled: true
+        accessModes:
+        - "ReadWriteOnce"
+        storage: "30Gi"
+
+  ## Use this to enable an extra service account
+  # serviceAccount:
+  #   annotations: {}
+  #   enabled: false
+  #   name: "sentry-clickhouse"
+  #   automountServiceAccountToken: true
+
+## This value is only used when clickhouse.enabled is set to false
+##
+externalClickhouse:
+  ## Hostname or ip address of external clickhouse
+  ##
+  host: "clickhouse"
+  tcpPort: 9000
+  httpPort: 8123
+  username: default
+  password: ""
+  database: default
+  ## Cluster name, can be found in config
+  ## (https://clickhouse.tech/docs/en/operations/server-configuration-parameters/settings/#server-settings-remote-servers)
+  ## or by executing `select * from system.clusters`
+  ##
+  # clusterName: test_shard_localhost
+
+# Settings for Kafka.
+# See https://github.com/bitnami/charts/tree/master/bitnami/kafka
+kafka:
+  enabled: true
+  replicaCount: 3
+  allowPlaintextListener: true
+  defaultReplicationFactor: 3
+  offsetsTopicReplicationFactor: 3
+  transactionStateLogReplicationFactor: 3
+  transactionStateLogMinIsr: 3
+  # 50 MB
+  maxMessageBytes: "50000000"
+  # 50 MB
+  socketRequestMaxBytes: "50000000"
+
+  service:
+    port: 9092
+
+  ## Use this to enable an extra service account
+  # serviceAccount:
+  #   create: false
+  #   name: kafka
+
+  ## Use this to enable an extra service account
+  # zookeeper:
+  #   serviceAccount:
+  #     create: false
+  #     name: zookeeper
+
+## This value is only used when kafka.enabled is set to false
+##
+externalKafka:
+  ## Hostname or ip address of external kafka
+  ##
+  # host: "kafka-confluent"
+  port: 9092
+
+redis:
+  enabled: true
+  auth:
+    enabled: false
+    sentinel: false
+  nameOverride: sentry-redis
+  usePassword: false
+  ## Just omit the password field if your redis cluster doesn't use password
+  # password: redis
+  master:
+    persistence:
+      enabled: true
+  ## Use this to enable an extra service account
+  # serviceAccount:
+  #   create: false
+  #   name: sentry-redis
+
+## This value is only used when redis.enabled is set to false
+##
+externalRedis:
+  ## Hostname or ip address of external redis cluster
+  ##
+  # host: "redis"
+  port: 6379
+  ## Just omit the password field if your redis cluster doesn't use password
+  # password: redis
+
+postgresql:
+  enabled: true
+  nameOverride: sentry-postgresql
+  postgresqlUsername: postgres
+  postgresqlDatabase: sentry
+  replication:
+    enabled: false
+    readReplicas: 2
+    synchronousCommit: "on"
+    numSynchronousReplicas: 1
+    applicationName: sentry
+  ## Use this to enable an extra service account
+  # serviceAccount:
+  #   enabled: false
+
+## This value is only used when postgresql.enabled is set to false
+##
+externalPostgresql:
+  # host: postgres
+  port: 5432
+  username: postgres
+  # password: postgres
+  database: sentry
+  # sslMode: require
```

Comment:
```
  File "/usr/local/lib/python3.6/site-packages/psycopg2/__init__.py", line 130, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
django.db.utils.OperationalError: FATAL:  password authentication failed for user "postgres"
```
Esse erro acima não se pode ser algum valor que não foi passado aqui.

Reply:
É sim falcão no caso esse valor seria do postgres, se for usar essa versão acho que vamos precisar de um externo